tracing-subscriber/src/filter/env/field.rs

```diff
@@ -41,7 +41,7 @@ pub(crate) enum ValueMatch {
     Bool(bool),
     U64(u64),
     I64(i64),
-    Pat(MatchPattern),
+    Pat(Box<MatchPattern>),
```

Comment:
Which clippy lint suggested this?

Reply:
I forget the name of the lint but it had to do with the size of the variant being unbalanced because `MatchPattern` was larger than the other fields. It suggested putting it behind a `Box` to make the variant smaller.

Reply:
Ah, yup, that makes sense. I _think_ this is good but I'll have to think it through.

Reply:
Here is the lint that suggested using the `Box`:

https://rust-lang.github.io/rust-clippy/master/index.html#large_enum_variant